.solr_wrapper.yml

```diff
@@ -1,6 +1,6 @@
 # Place any default configuration for solr_wrapper here
 version: 8.8.2
-solr_zip_path: tmp/solr_install
+download_dir: tmp
```

Comment:
I think for this syntax to work, we need to upgrade to a 4.x version of solr_wrapper.

Reply:
@mark-dce Thanks for catching that as a potential problem. I actually got that option from the README on the 3.1.2 tag: https://github.com/cbeer/solr_wrapper/blob/v3.1.2/README.md .  I tested both 3.1.2 and 4.x and both worked, but if this is working for you all I will just leave it alone for now.  I will probably change it at some point in the future since `solr_zip_path` seems to have been dropped in 4.x.